### ISSUE TITLE
Sorting doesn't work properly

### ISSUE THREAD

**Issue by alexandernst** [atom/tree-view#814]
https://ibin.co/2eRYRMpZLgxi.png

I'd expect folders to be sorted alphabetically (as they do in upper directories), but for some reason the `assets` folder in a Yii2 framework project is making the sorting behave in a strange manner.

**Comment by thomasjo** [comment]
Are you able to reproduce this in safe mode? I just want to rule out the possibility this is somehow caused by a community package or some such.

**Comment by alexandernst** [comment]
@thomasjo Yes, this also happens with `--safe`. In fact, this is as easy to repro as creating 3 folders:

```
a61c7cb
a69e85b9
a6264eb9
```

**Comment by lee-dohm** [comment]
This is most likely caused by the natural sorting algorithm added in https://github.com/atom/tree-view/pull/621. It may need to have an exception added for hexadecimal numbering?

**Comment by alexandernst** [comment]
Were you able to repro this bug? Do you need me to provide further info?

**Comment by 50Wliu** [comment]
Confirmed on Windows 10 x64, Atom 1.9.0-dev-6b31780.

The folders are displayed in the same order as in @alexandernst's comment.

**Comment by 50Wliu** [comment]
I'm seeing similar behavior in Windows Explorer:
![image](https://cloud.githubusercontent.com/assets/2766036/14756326/6a5c15e6-08b6-11e6-95bb-e6dd563faee3.png)

**Comment by alpaco** [comment]
Hi. I am running into the exact same issue: filenames containing hex numbers get mixed up. Natural sort algorithm is a good thing. It just doesn't work with hex numbers.

Why? The algorithm can't know whether we are using decimal numbers and revisions, or hex numbers. Example:

```
DEC NOs & REVs      HEX NOs
(correct sorting)   (incorrect sorting)
=================   ===================
chapter01.doc       mem_bank_01.asm
chapter01a.doc      mem_bank_1A.asm
chapter1b.doc       mem_bank_1B.asm
chapter2.doc        mem_bank_02.asm
chapter09.doc       mem_bank_09.asm
```

So in my particular project I would like the right column to be sorted correctly. The particular commit (#621) tells us:

> The commit also adds an option to use the old sort function, incase this one doesnt work with other locales, or if users prefer posix style sorting.

Yea, that's me, thank you! ;-) I can't find the option however. I might be looking at the wrong place. FYI: I was expecting to find it somewhere over here:

![afbeelding](https://user-images.githubusercontent.com/21052354/78414460-68584a80-761c-11ea-8907-febb54e7ff31.png)

Kind regards!
